docs/release-notes/8.0.40-31.md

```diff
@@ -0,0 +1,37 @@
+# Percona Server for MySQL 8.0.40-31 (2024-11-08)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.40-31](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the [MySQL 8.0.40 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-40.html) and enterprise-grade features developed by Percona.
```

Comment:
[Percona Server for MySQL 8.0.40-31](https://www.percona.com/software/mysql-database/percona-server) - The page you requested could not be found.